compiler/src/dotty/tools/dotc/fromtasty/JavaCompilationUnit.scala

```diff
@@ -0,0 +1,13 @@
+package dotty.tools.dotc.fromtasty
+
+import dotty.tools.dotc.CompilationUnit
+import dotty.tools.dotc.util.NoSource
+
+/**
+ * A marker CompilationUnit to return up the call stack from ReadTasty.  This will tell us that we've
+ * encountered, and attempted to inspect, a Java class file.  We can't TASTy-inspect a Java class obviously,
+ * but we want to return the fact we found it so that higher-up we can take appropriate action if desired.
+ */
```

Comment:
Nitpick: we format the comments as

```
/** A marker CompilationUnit to return up the call stack from ReadTasty.  This will tell us that we've
 *  encountered, and attempted to inspect, a Java class file.  We can't TASTy-inspect a Java class obviously,
 *  but we want to return the fact we found it so that higher-up we can take appropriate action if desired.
 */
```

Reply:
This is good feedback.  Will work on these changes--and the comments.

Reply:
A question:  A "normal" CompilationUnit does not currently store the full class name.  The JavaCompilationUnit and Scala2CompilationUnit flavors do.  What should compilationUnitClassname() return for a Scala3/Tasty CompilationUnit?  Some options:
1) Return "" for normal CU
2) Make compilationUnitClassname() return Option[String]

Any preferences?